enos/enos-variables.hcl

```diff
@@ -79,6 +79,7 @@ variable "target_count" {
 variable "local_boundary_dir" {
   description = "Path to local boundary executable"
   type        = string
+  default     = "/tmp"
```

Comment:
We could maybe do away with this as it was part of trying to figure out a previous issue. It's not harming anything to be here.
```suggestion
```

Reply:
I would lean towards removing it if it's not used for anything.